vagrant/provisioning/roles/arkcase-ldap-config/templates/spring-config-ldap-pre-3.3.3.xml

```diff
@@ -0,0 +1,217 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans:beans xmlns="http://www.springframework.org/schema/security" xmlns:beans="http://www.springframework.org/schema/beans"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.springframework.org/schema/beans 
+    http://www.springframework.org/schema/beans/spring-beans-4.1.xsd
+    http://www.springframework.org/schema/security http://www.springframework.org/schema/security/spring-security-3.2.xsd">
```

Comment:
same as the old template

Reply:
We would need an openldap/apacheds template. The new AD provider won't work correctly on these LDAP servers.

Reply:
Let's not add OpenLDAP support to the installer until a customer requires it... given our customer base I would be surprised to find anybody running on OpenLDAP.  For the open source guys, Samba should work well.